html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email:</label> <br>
+        <input type="text" name="email">
+      </div>
+      <div>
+        <label for="cool">Why is coding cool?</label> <br>
+        <select name="cool">
+          <option value="option">Select an option</option>
+          <option value="interest">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="power">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="language">Favorite language (choose one)</label> <br>
```

Comment:
This can just be general text, it isn't labelling a specific form control element